joomla/joomla-cms
# Allow to disable js/css assets auto-reload in system debug plugin

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/16381.

### Summary of Changes

Adds an option to the debug system plugin to allow to disable the js/css assets auto-reload on every page load in system debug plugin.

### Testing Instructions

1. Apply patch
2. Enable debug system plugin and debug in global config
3. Check there is a media version hash in assets (js/css) in the page source code
3. Now, in the debug system plugin disable the new "Refresh Assets" option
4. Save, reload the page and check there is no media version hahs in assets (js/css) in the page source code

### Expected result

Allow to disable js/css assets auto-reload in debug system plugin.

### Actual result

Not allowed to disable js/css assets auto-reload in debug system plugin.

### Documentation Changes Required

none.
